README.md

```diff
@@ -7,29 +7,67 @@ This is a container which will be called from:
 1. Canary Tests
 2. Regular Build PRs
 
-This will be called after Observability is installed - both Hub and Addon in real OCP clusters or Kind.
-
 The tests in this container will:
 
 1. Create the MCO CR . The Object store to be already in place for CR to work.
 2. Wait for the installation to complete.
 3. Then check the entire Observability suite (Hub and Addon) is working as expected including disable/enable, Grafana etc.
 
-## Running E2E
+## Setup E2E Testing Environment
+
+If you only have an OCP cluster and haven't installed Observability yet, then you can install the Observability(both Hub and Addon) by the following steps:
```

Comment:
space is needed. (`Observability(both Hub and Addon)`)

Reply:
Done.